package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "server-training",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
add script for develop - use ts-node (it will help you a lot during development)

Reply:
done